src/shared/components/socialLogin/socialMediaButtons.css

```diff
@@ -0,0 +1,73 @@
+/* Credit: https://codepen.io/davidelrizzo/pen/vEYvyv */
```

Comment:
These were mentioned in Slack by @nagano564:

* There should be a `cursor: pointer` rule on these buttons, similar to our other buttons.
* The hover color change on the Google button is so subtle is it barely noticeable - perhaps exaggerate it a bit